### ISSUE TITLE
tensorboard embedding projector not working

### ISSUE THREAD

**Issue by sgbaird** [faris-k/fastsiam-wafers#1]
- https://github.com/tensorflow/tensorboard/issues/3840
- [Tensorboard parsing metadata or fetching sprite images takes forever](https://stackoverflow.com/questions/44096492/tensorboard-parsing-metadata-or-fetching-sprite-images-takes-forever)

**Comment by faris-k** [comment]
@sgbaird psssst [check this out](https://projector.tensorflow.org/?config=https://gist.githubusercontent.com/faris-k/658a3bc7eda3010804eb886ecb93744d/raw/7e951051224c0d6683a790d928e887a90a80ebdd/fiastsiam_1_config.json)

## The issue
Access to sprite image was blocked by CORS policy

![image](https://user-images.githubusercontent.com/68213622/211691685-4be13e50-c8f1-425a-a875-f760212a6782.png)

## The fix
Epic googling of "TensorBoard projector CORS policy" leads to a fix [here](https://github.com/tensorflow/tensorflow/issues/16770#issuecomment-367115996)

Broken config file:
```json
{
  "embeddings": [
    {
      "tensorName": "Wafer Embeddings",
      "tensorShape": [
        623,
        512
      ],
      "tensorPath": "https://raw.githubusercontent.com/faris-k/fastsiam-wafers/main/embedding_projector/feature_vecs.tsv",
      "metadataPath": "https://raw.githubusercontent.com/faris-k/fastsiam-wafers/main/embedding_projector/failure_types.tsv",
      "sprite": {
        "imagePath": "https://github.com/faris-k/fastsiam-wafers/blob/main/embedding_projector/sprite.png?raw=true",
        "singleImageDim": [
          128,
          128
        ]
      }
    }
  ]
}
```

Fixed config file (literally one line, the `imagePath` 🙃)
```json
{
  "embeddings": [
    {
      "tensorName": "Wafer Embeddings",
      "tensorShape": [
        623,
        512
      ],
      "tensorPath": "https://raw.githubusercontent.com/faris-k/fastsiam-wafers/main/embedding_projector/feature_vecs.tsv",
      "metadataPath": "https://raw.githubusercontent.com/faris-k/fastsiam-wafers/main/embedding_projector/failure_types.tsv",
      "sprite": {
        "imagePath": "https://raw.githubusercontent.com/faris-k/fastsiam-wafers/main/embedding_projector/sprite.png",
        "singleImageDim": [
          128,
          128
        ]
      }
    }
  ]
}
```

## TL;DR

Make sure everything in your config file points to `githubusercontent.com` instead of `github.com` or something else.
